rmgpy/tools/fluxdiagram.py

```diff
@@ -321,7 +321,6 @@ def simulate(reactionModel, reactionSystem, settings=None):
     time = []
     coreSpeciesConcentrations = []
     coreReactionRates = []
-    edgeReactionRates = []
```

Comment:
Well the current code only loads a single Chemkin file, so `reactionModel.edge.species` and `reactionModel.edge.reactions` will be empty. I removed these lines because they weren't actually being used in the rest of the code. I guess I could add functionality to load an edge Chemkin file as well, but maybe that's better left for a future PR.

Reply:
Sorry, that was an old comment from before you reformatted the PR.